docs/content/Schema/Fundamentals/Working-with-Joins.mdx

```diff
@@ -730,233 +741,103 @@ cubes:
 
 </CodeTabs>
 
-The first case is to calculate the total revenue per customer. Let's name this
-metric `total_revenue`. We also need to be aware of the fact that orders can be
-placed without customer registration (anonymous customers/guest checkouts).
-Because of anonymous customers, we should start the join from the `orders` onto
-the `customers` cube so that we do not lose data from anonymous orders:
+With the given data model, we have two valid analytics use cases that require different join directions.
 
-<CodeTabs>
+The first case is to calculate the total revenue per customer. 
+To do this, we'll use the `total_revenue` measure that is defined on the orders cube. 
+We need to be aware that orders can be placed without customer registration (anonymous customers/guest checkouts). 
+Therefore, we should start the join from the `orders` cube onto the `customers` cube to ensure that we do not lose data from anonymous orders.
```

Comment:
```suggestion
Therefore, we should ensure that the `orders` cube is on the _left_ side of the join so we do not lose data from anonymous orders.
```
I'm not sure that "start the join" is a good way to put it. Also, in the beginning of this particular page, "_left_" semantics is already used (please check it out).